front-end-side/src/Pages/Data.js

```diff
@@ -71,102 +71,112 @@ const EnterData = () => {
+                                {allShown ?
+                                    <button type="button" onClick={showLess}>Show less</button>
+                                    : <button type="button" onClick={showAll}>Show more</button>
+                                }
+                            </div>
+                            <div className="container">
 
-                <h2 className="title">Statistics</h2>
-                {
-                    (statusForExpenses && categoryStatus) ? (
-                        <div id="statistics">
-                            <div id="cards">
-                                <ExpensesCard name="food" categorySum={categoryData.data.foodSum} expenses={newExpenses} />
-                                <ExpensesCard name="transportation" categorySum={categoryData.data.carSum} expenses={newExpenses} />
-                                <ExpensesCard name="entertainment" categorySum={categoryData.data.entertaintmentSum} expenses={newExpenses} />
-                                <ExpensesCard name="house" categorySum={categoryData.data.houseSum} expenses={newExpenses} />
-                                <ExpensesCard name="clothes" categorySum={categoryData.data.clothesSum} expenses={newExpenses} />
-                                <ExpensesCard name="other" categorySum={categoryData.data.otherSum} expenses={newExpenses} />
+                                <h2 className="title">Statistics</h2>
+                                {(statusForExpenses && categoryStatus) ? (
+                                    <div id="statistics">
+                                        <div id="cards">
+                                            <ExpensesCard name="food" categorySum={categoryData.data.foodSum} expenses={newExpenses} />
+                                            <ExpensesCard name="transportation" categorySum={categoryData.data.carSum} expenses={newExpenses} />
+                                            <ExpensesCard name="entertainment" categorySum={categoryData.data.entertaintmentSum} expenses={newExpenses} />
+                                            <ExpensesCard name="house" categorySum={categoryData.data.houseSum} expenses={newExpenses} />
+                                            <ExpensesCard name="clothes" categorySum={categoryData.data.clothesSum} expenses={newExpenses} />
+                                            <ExpensesCard name="other" categorySum={categoryData.data.otherSum} expenses={newExpenses} />
+                                        </div>
+                                    </div>
+                                ) : (
+                                    null
+                                )}
                             </div>
-                        </div>
+                        </>
                     ) : (
-                        null
+                        <>
+                            <div className="container" style={{ marginTop: "30px" }}>
```

Comment:
This block (modals etc.) is repeated, isn't it possible to make this update without repeating code?

Reply:
Fixed it